phpunit/class-edit-site-export-test.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Test `gutenberg_edit_site_export` and its helper functions.
+ *
+ * @package    Gutenberg
+ */
+
+class Edit_Site_Export_Test extends WP_UnitTestCase {
+	function test_inject_theme_attribute_in_content() {
```

Comment:
This should be `test_remove_theme_attribute_from_content`

Reply:
👍